h2o-py/h2o/model/multinomial.py

```diff
@@ -1,13 +1,13 @@
 # -*- encoding: utf-8 -*-
 from __future__ import absolute_import, division, print_function, unicode_literals
-from h2o.utils.compatibility import *  # NOQA
 
 import h2o
+from h2o.utils.compatibility import *  # NOQA
 from h2o.utils.typechecks import assert_is_type
-from ..exceptions import H2OValueError
-from ..frame import H2OFrame
 from .extensions import has_extension
 from .model_base import ModelBase
+from ..exceptions import H2OValueError
+from ..frame import H2OFrame
```

Comment:
actually according to my rule: top-down from furthest to closest, this was better before ;)

(the rationale is: if there is a name conflict and one name is overridden by mistake, then the closest should take precedence to follow usual scoping rules, and make it easier to identify the conflict)